frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.unit.spec.tsx

```diff
@@ -1,43 +1,67 @@
 import { checkNotNull } from "metabase/core/utils/types";
 import { getMetadata } from "metabase/selectors/metadata";
-import { SchemaId, TableId } from "metabase-types/api";
+import { InitialSyncStatus, Schema, Table } from "metabase-types/api";
 import { createMockDatabase, createMockTable } from "metabase-types/api/mocks";
 import { createMockState } from "metabase-types/store/mocks";
 import { createMockEntitiesState } from "__support__/store";
-import { renderWithProviders, screen } from "__support__/ui";
+import { renderWithProviders, screen, within } from "__support__/ui";
 import DataSelectorTablePicker from "./DataSelectorTablePicker";
 
 const TEST_TABLE = createMockTable();
 const TEST_DATABASE = createMockDatabase({ tables: [TEST_TABLE] });
+const NOT_SYNCED_DB_STATUSES: InitialSyncStatus[] = ["aborted", "incomplete"];
 
 interface SetupOpts {
-  schemaIds?: SchemaId[];
-  tableIds?: TableId[];
+  schemas?: Schema[];
+  tables?: Table[];
 }
 
-const setup = ({ schemaIds = [], tableIds = [] }: SetupOpts = {}) => {
+const setup = ({ schemas = [], tables = [] }: SetupOpts = {}) => {
+  const TEST_DATABASE = createMockDatabase({ tables: tables });
```

Comment:
nit: we could use shorthand syntax
```suggestion
  const TEST_DATABASE = createMockDatabase({ tables });
```

Reply:
I've replaced this part now